graylog2-web-interface/src/routing/AppRouter.jsx

```diff
@@ -162,12 +165,15 @@ const AppRouter = () => {
                      component={EditContentPackPage} />
               <Route path={Routes.SYSTEM.CONTENTPACKS.show(':contentPackId')} component={ShowContentPackPage} />
               <Route path={Routes.SYSTEM.GROKPATTERNS} component={GrokPatternsPage} />
-              <Route path={Routes.SYSTEM.INDICES.LIST} component={IndicesPage} />
-              <Route path={Routes.SYSTEM.INDEX_SETS.CREATE} component={IndexSetCreationPage} />
-              <Route path={Routes.SYSTEM.INDEX_SETS.SHOW(':indexSetId')} component={IndexSetPage} />
-              <Route path={Routes.SYSTEM.INDEX_SETS.CONFIGURATION(':indexSetId')}
-                     component={IndexSetConfigurationPage} />
-              <Route path={Routes.SYSTEM.INDICES.FAILURES} component={IndexerFailuresPage} />
+
+              {!isCloud && <Route path={Routes.SYSTEM.INDEX_SETS.CREATE}
+                                  component={IndexSetCreationPage} />}
+              {!isCloud && <Route path={Routes.SYSTEM.INDEX_SETS.SHOW(':indexSetId')}
+                                  component={IndexSetPage} />}
+              {!isCloud && <Route path={Routes.SYSTEM.INDEX_SETS.CONFIGURATION(':indexSetId')}
```

Comment:
These three lines have an eslint error:

`ESLint: Missing parentheses around multilines JSX(react/jsx-wrap-multilines)`

Reply:
Good catch. I improved that part and grouped the routes into one fragment